src/main/java/Task.java

```diff
@@ -0,0 +1,22 @@
+public class Task {
+    private String done;
```

Comment:
Should the indicator of whether a Task is done be a boolean?
```suggestion
    private boolean isDone;
```